src/app/components/ms-contents/ms-contents.component.html

```diff
@@ -1,54 +1,50 @@
-            </ng-container>
-            <ng-container *ngIf="data?.textLangs.length>0">
-                <div class="flex-item-label">
-                    <evt-ms-desc-section [label]="'textLang'" [inlineLabel]="true"></evt-ms-desc-section>
-                </div>
-                <div class="flex-item-information">
-                    <div>
-                        <span *ngFor="let textLangs of data.textLangs; let i = index">
-                            <evt-content-viewer *ngFor="let textLang of textLangs" [content]="textLang"></evt-content-viewer>
-                            <span *ngIf="i < [data.textLangs.length-1]">, </span>
-                        </span>
-                    </div>
-                </div>
-            </ng-container>
-            <ng-container *ngIf="data?.msItem.length>0">
-                <div class="flex-item-label">
-                    <evt-ms-desc-section [label]="'items'" [inlineLabel]="true"></evt-ms-desc-section>
+    <div class="flex-container">
+        <ng-container *ngIf="data?.summary">
+            <div class="flex-item-label">
+                <evt-ms-desc-section [label]="'summary'" [inlineLabel]="true"></evt-ms-desc-section>
+            </div>
+            <div class="flex-item-information">
+                <evt-content-viewer [content]="data.summary"></evt-content-viewer>
+            </div>
+        </ng-container>
+        <ng-container *ngIf="data?.textLangs.length>0">
+            <div class="flex-item-label">
+                <evt-ms-desc-section [label]="'textLang'" [inlineLabel]="true"></evt-ms-desc-section>
+            </div>
+            <div class="flex-item-information">
+                <div>
+                    <span *ngFor="let textLangs of data.textLangs; let i = index">
+                        <evt-content-viewer *ngFor="let textLang of textLangs" [content]="textLang"></evt-content-viewer>
+                        <span *ngIf="i < [data.textLangs.length-1]">, </span>
+                    </span>
                 </div>
-            </ng-container>
-        </div>
-        <ng-container *ngFor="let m of data?.msItem">
-            <evt-ms-item [data]="m"></evt-ms-item>
-            <ng-container *ngIf="m.msItem?.length>0">
-                <evt-ms-item *ngFor="let ms of m.msItem" [data]="ms" [nested1]="true"></evt-ms-item>
-                <ng-container *ngFor="let ms of m.msItem">
-                    <evt-ms-item *ngFor="let item of ms.msItem" [data]="item" [nested2]="true"></evt-ms-item>
-                </ng-container>
+            </div>
+        </ng-container>
+        <ng-container *ngIf="data?.msItem.length>0">
+            <div class="flex-item-label">
+                <evt-ms-desc-section [label]="'items'" [inlineLabel]="true"></evt-ms-desc-section>
+            </div>
+        </ng-container>
+    </div>
+    <ng-container *ngFor="let m of data?.msItem">
+        <evt-ms-item [data]="m"></evt-ms-item>
+        <ng-container *ngIf="m.msItem?.length>0">
```

Comment:
This check is not necessary. If there is no msItem the array will be zero length.